packages/blade/docs/guides/Intro.stories.mdx

```diff
@@ -33,7 +33,7 @@ Blade is the Cross-Platform, Open Source Design System that powers all of the [R
 
 <div style={{ textAlign: 'left' }}>
   <iframe
-    width="560"
```

Comment:
<img width="1102" alt="image" src="https://github.com/user-attachments/assets/b753d508-b832-43e3-ac33-15664bd719e5">

Wouldn't this break it on desktop?

Reply:
This is how it looks on desktop. It takes 100% of the screen. Is this ok @saurabhdaware ?
<img width="1072" alt="Screenshot 2024-10-17 at 11 19 52 AM" src="https://github.com/user-attachments/assets/f5ea4813-847f-42b2-9978-cef1bab4c5cb">

If not what do you think of adding overflow:auto instead of setting the width to 100%.

Reply:
We can have a defined width with maxWidth as 100%. In that case it won't overflow in mobile but still have expected width on desktop

Reply:
Thanks for the suggestion. I have updated the changes. @saurabhdaware